### PR TITLE
fix(TDOPS-4964/components): SidePanel colors

### PR DESCRIPTION
**Pull request by inna-i**
**What is the problem this PR is trying to solve?**
[TDOPS-4964](https://jira.talendforge.org/browse/TDOPS-4964): Side Menu colors

**What is the chosen solution to this problem?**
- SidePanel color fixed for different states
- unused bootstrap vars removed
image

**Please check if the PR fulfills these requirements**

- [x] The PR have used `yarn changeset` to a request a release from the CI if wanted.
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: fork/bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss [hunk 1]
```diff
@@ -441,12 +441,6 @@ $nav-tabs-active-link-hover-border-color:   #ddd !default;
 $nav-tabs-justified-link-border-color:            #ddd !default;
 $nav-tabs-justified-active-link-border-color:     $body-bg !default;
 
-//== Pills
-$nav-pills-border-radius:                   $border-radius-base !default;
```
Comment (inna-i) [inline fork/bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss:445]: Not needed

File: packages/components/src/ActionList/ActionList.component.js [hunk 2]
```diff
@@ -89,7 +89,7 @@ function ActionList(props) {
 				'tc-action-list',
 				className,
 				{
-					'nav-inverse': props.reverse,
+					'nav-inverse': !props.reverse,
```
Comment (inna-i) [inline packages/components/src/ActionList/ActionList.component.js:92]: To have the same condition, as in SidePanel

File: packages/components/src/SidePanel/SidePanel.component.js [hunk 3]
```diff
@@ -101,9 +101,7 @@ function SidePanel({
 		[theme.reverse]: reverse,
 		[theme.animate]: animation,
 	});
-	const listCSS = classNames(theme['tc-side-panel-list'], 'tc-side-panel-list', {
-		'nav-inverse': !reverse,
```
Comment (inna-i) [inline packages/components/src/SidePanel/SidePanel.component.js:105]: Duplicated

File: .changeset/real-needles-study.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+'@talend/bootstrap-sass': patch
```
Comment (Gbacc) [inline .changeset/real-needles-study.md:2]: Was wondering if we were supposed to change this package
Seems like it's a fork for technical reasons but if anything needs to change , it should be an override in @talend/bootstrap-theme package

Comment (inna-i) [reply .changeset/real-needles-study.md:2]: It was generated by the command `yarn changeset`. Maybe you are right and this update is not needed :thinking:

Comment (inna-i) [reply .changeset/real-needles-study.md:2]: Gonna revert part of changes

File: packages/theme/src/theme/_navs.scss [hunk 5]
```diff
@@ -33,12 +33,24 @@ $tc-tabbar-active-border-size: 0.2rem !default;
...
+				}
+			}
+
 			&.active,
 			&.open {
 				.btn-link {
-					color: tokens.$coral-color-accent-text-weak;
-					background-color: tokens.$coral-color-accent-background-strong;
+					background-color: tokens.$coral-color-accent-text-weak;
+					color: tokens.$coral-color-accent-background-strong;
```
Comment (Gbacc) [inline packages/theme/src/theme/_navs.scss:53]: This is strange to have a text color for a background and a background color for text 😄

Comment (inna-i) [reply packages/theme/src/theme/_navs.scss:53]: Right :smile: It was changed quickly for a quick check. I can find correct tokens

**Review by Gbacc — APPROVED** [review]

**Review by VolodymyrKovalM — APPROVED** [review]

**Pull request merged**